src/routes/results.js

```diff
@@ -40,14 +42,7 @@ class Results extends React.Component {
             style={styles.icon}
             source={icon}
             />
-            <LargeText
-            style={styles.headerUpper}
-            text = {"Material: " + label}
-            />
-            <LargeText
-            style={styles.headerUpper}
-            text = {"Probability: " + probability}
-            />
+            getJSX(URLindexToObj(getURLindex(label)))
```

Comment:
typically in packaged javascript you need to export then import these to reference them in another file... but if you are able to access them here maybe react native works differently. 
In a real world app it is (almost) always a terrible idea to use global anything.